training-node/test/users.js

```diff
@@ -191,6 +192,24 @@ describe('users', () => {
           done();
         });
     });
+    it('should fail authorization because of expiration token', done => {
+      config.common.session.expiration = 1;
+      exports.successAdminAuth().then(response => {
+        const token = response.headers[tokenManager.HEADER_NAME];
+        setTimeout(() => {
+          chai
+            .request(server)
+            .get('/users')
+            .set(tokenManager.HEADER_NAME, token)
+            .catch(err => {
+              err.response.should.have.status(401);
+              err.response.body.should.have.property('error');
+              done();
+              config.common.session.expiration = 300;
```

Comment:
why this?

Reply:
Because other test breaks without that, i will fix that and move config to test environment